core/src/services/memcached/backend.rs

```diff
@@ -82,6 +87,18 @@ impl MemcachedBuilder {
         self.config.default_ttl = Some(ttl);
         self
     }
+
+    /// Set the tls connect on.
+    pub fn tls(mut self, tls: bool) -> Self {
+        self.config.tls = Some(tls);
+        self
+    }
+
+    /// Set the tls connect on.
+    pub fn cafile(mut self, cafile: PathBuf) -> Self {
```

Comment:
I believe we should provide two options here:

- We should load users' CA by default (via `rustls_native_certs`).
- We should provide `key` and `cert` otpion for users to use their own cert.